QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -71,8 +92,29 @@ public void event() {
 			leftBackMotor.set(leftBack);
 		}
 	};
+	
+	private static EventOutput calibrate = new EventOutput() {
+		public void event() {
+			calibratedAngle.set((float) (HeadingSensor.yaw.get() / 180 * π));
+			Logger.info("Calibrated Angle: " + HeadingSensor.yaw.get());
```

Comment:
When you report the angle that we calibrated to, you should make sure that it's the same one you set - don't read the value twice because then it might have changed.